user.go

```diff
@@ -0,0 +1,101 @@
+package myradio
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"time"
+)
+
+type Officership struct {
+	OfficerId   uint   `json:"officerid,string"`
+	OfficerName string `json:"officer_name"`
+	TeamId      uint   `json:"teamid,string"`
+	FromDateRaw string `json:"from_date,omitempty"`
+	FromDate    time.Time
```

Comment:
The long term solution is to refactor the API to return all datetimes in a format that go handles without all this fuss :smile: